tests/foreman/api/test_syncplan.py

```diff
@@ -196,9 +209,13 @@ def test_positive_create_with_description(module_org, description):
 
     :CaseImportance: Critical
     """
-    sync_plan = entities.SyncPlan(description=description, organization=module_org).create()
+    sync_plan = entities.SyncPlan(
+        enabled=False, description=description, organization=module_org
+    ).create()
     sync_plan = sync_plan.read()
     assert sync_plan.description == description
+    # disable sync plan after test
+    disable_syncplan(sync_plan)
```

Comment:
This is already `enabled=False` Syncplan, need not to disable again.

Reply:
OK, fixed